tsl/test/shared/sql/constify_now.sql.in

```diff
@@ -8,81 +8,84 @@ SET timescaledb.current_timestamp_mock TO '1990-01-01';
 
 \set PREFIX 'EXPLAIN (COSTS OFF, SUMMARY OFF, TIMING OFF)'
 
+-- create a test table
+-- any query with successful now_constify will have 1 chunk while
+-- others will have 2 chunks in plan
+CREATE TABLE const_now(time timestamptz, time2 timestamptz, device_id int, value float);
+SELECT table_name FROM create_hypertable('const_now','time');
+INSERT INTO const_now SELECT '1000-01-01','1000-01-01',1,0.5;
+INSERT INTO const_now SELECT '1000-01-01','1000-01-01',2,0.5;
+INSERT INTO const_now SELECT '3000-01-01','3000-01-01',1,0.5;
+INSERT INTO const_now SELECT '3000-01-01','3000-01-01',2,0.5;
+
 -- test valid variants we are optimizing
 -- all of these should have a constified value as filter
 -- none of these initial tests will actually exclude chunks
 -- because we want to see the constified now expression in
 -- EXPLAIN output
-:PREFIX SELECT FROM metrics WHERE time > now();
-:PREFIX SELECT FROM metrics WHERE time >= now();
-:PREFIX SELECT FROM metrics WHERE time > now() - '24h'::interval;
-:PREFIX SELECT FROM metrics WHERE time > now() + '10m'::interval;
-:PREFIX SELECT FROM metrics WHERE time >= now() - '10m'::interval;
-:PREFIX SELECT FROM metrics WHERE time >= now() + '10m'::interval;
+:PREFIX SELECT FROM const_now WHERE time > now();
+:PREFIX SELECT FROM const_now WHERE time >= now();
+:PREFIX SELECT FROM const_now WHERE time > now() - '24h'::interval;
+:PREFIX SELECT FROM const_now WHERE time > now() + '10m'::interval;
+:PREFIX SELECT FROM const_now WHERE time >= now() - '10m'::interval;
+:PREFIX SELECT FROM const_now WHERE time >= now() + '10m'::interval;
+:PREFIX SELECT FROM const_now WHERE time > now() - '2d'::interval;
```

Comment:
would be good to setup a test to show that expected chunks are excluded when DST change happens.
set mock time = DST time (for a specific timezone) + chunks with 1 day intervals  (with chunks for day before DST change/on day of DST change/after DST change).

Reply:
done